nodogsplash/Makefile

```diff
@@ -40,19 +40,27 @@ define Package/nodogsplash/description
 endef
 
 define Package/nodogsplash/install
-	$(CP) ./files/* $(1)/
+	$(INSTALL_DIR) $(1)/usr
+	$(INSTALL_DIR) $(1)/usr/lib
```

Comment:
I think, we can drop this, right? I bet that INSTALL_DIR is recursive, so leaving only /usr/lib/donogsplash should be enough. Same applies for /etc.

Otherwise, good job! :) I feel like we are reaching the end.

Reply:
Thank you for you patience. I have also removed most of the needless INSTALL_DIR calls.

Reply:
Looks good now! Thank you and sorry that it took so long.